File: solita-dev-academy-2023-server/Models/Journey.cs
namespace solita_dev_academy_2023_server.Models
{
    [Serializable]
    public class Journey
    {
        public DateTime? Departure { get; set; }
        public DateTime? Return { get; set; }
        public string? Departure_station_name_fi { get; set; }
        public string? Departure_station_name_se { get; set; }
        public string? Departure_station_name_en { get; set; }
        public string? Departure_station_address_fi { get; set; }
        public string? Departure_station_address_se { get; set; }
        public string? Return_station_name_fi { get; set; }
        public string? Return_station_name_se { get; set; }
        public string? Return_station_name_en { get; set; }
        public string? Return_station_address_fi { get; set; }
        public string? Return_station_address_se { get; set; }
        public double? Covered_distance { get; set; }
        public float? Duration { get; set; }
    }
}

File: solita-dev-academy-2023-client/src/Models/Journey.ts
export default interface Journey {
    departure: string;
    return: string;
    departure_station_name_fi: string;
    departure_station_name_se: string;
    departure_station_name_en: string;
    departure_station_address_fi: string;
    departure_station_address_se: string;
    return_station_name_fi: string;
    return_station_name_se: string;
    return_station_name_en: string;
    return_station_address_fi: string;
    return_station_address_se: string;
    covered_distance: number;
    duration: number;
}